private/bufpkg/bufplugin/bufplugindocker/docker_test.go

```diff
@@ -180,7 +197,7 @@ func buildDockerPlugin(t testing.TB, dockerClient Client, dockerfilePath string,
 	require.Nilf(t, err, "failed to open dockerfile")
 	pluginName, err := bufpluginref.PluginIdentityForString(pluginIdentity)
 	require.Nilf(t, err, "failed to create plugin identity")
-	pluginConfig := &bufpluginconfig.Config{Name: pluginName}
+	pluginConfig := &bufpluginconfig.Config{Name: pluginName, PluginVersion: "v1.0.0"}
```

Comment:
Should this be an arg we pass to the func?

Reply:
Good idea - updated.